### ISSUE TITLE
File not found when generating PDF

### ISSUE THREAD

**Issue by edigonzales** [claeis/oereb-web-service#13]
Tried `/tmp` and `/home/oereb-web-service`

```
Sat Aug 17 16:25:35 GMT 2019
There was an unexpected error (type=Internal Server Error, status=500).
JAXB marshalling exception; nested exception is javax.xml.bind.MarshalException - with linked exception: [java.io.FileNotFoundException: /home/oereb-web-service,/oerebws18/CH540632893416.xml (No such file or directory)]
```

Is this just the message or is there really a comma added?

**Comment by edigonzales** [comment]
Log output: 
```
web-service_1  | 2019-08-17 16:31:14.554  INFO 6 --- [nio-8181-exec-1] ch.ehi.oereb.webservice.OerebController  : tmpFolder /home/oereb-web-service,/oerebws18
```

**Comment by claeis** [comment]
fixed in b12ce9f79771039889364505aa0d23764a2db111
